tasks/configuration.yml

```diff
@@ -100,6 +103,7 @@
     - services.conf
     - templates.conf
     - commands.conf
+  when: inventory_hostname == icinga2_client_monitoring_parents[0]
```

Comment:
Is it possible to reorder those tasks thus all tasks which have the `when: inventory_hostname == icinga2_client_monitoring_parents[0]` condition can be put in one big `block` section? To my knowledge this would skip the whole block if not required and could optimize run-time.

Reply:
`ERROR! 'loop' is not a valid attribute for a Block`
Need to check if I can solve this somehow